examples/zkapps/04-zkapp-browser-ui/ui/pages/_app.page.tsx

```diff
@@ -59,7 +59,7 @@ export default function App() {
         console.log('zkApp compiled');
 
         const zkappPublicKey = PublicKey.fromBase58(
-          'B62qiqD8k9fAq94ejkvzaGEV44P1uij6vd6etGLxcR4dA8ZRZsxkwvR'
+          'B62qohp4zipB5jHp2r8tZyCX1j65H37aQjn4vh9uyHskn3jfnSxbeRu'
```

Comment:
Why are we changing public keys here?

Reply:
The contract was deployed a long time ago. I decided to deploy a new version of the Add example contract for this tutorial to interact with.